src/Zinc-FileSystem/package.st

```diff
@@ -1 +0,0 @@
-Package { #name : #'Zinc-FileSystem' }
```

Comment:
Is this removal ok?

Reply:
Yes, for now we keep an empty package because the bootstrap fails without it.
I’m trying to remove the references to it in the bootstrap to not need it anymore. 
but from your second comment, this still needs work